conf/messages.en

```diff
@@ -183,6 +183,15 @@ reportDateRange.checkYourAnswersLabel = What date range do you want the report t
 reportDateRange.error.required = Select the date range you want the report to cover
 reportDateRange.change.hidden = Report Date Range
 
+reportName.title = report Name
+reportName.heading = What name will you use to identify this report?
+reportName.checkYourAnswersLabel = What name will you use to identify this report?
+reportName.hint = This name is only used to help you manage your reports
+reportName.error.required = Enter the name you will use to identify this report
+reportName.error.length = ReportName must be 70 characters or less
+reportName.error.invalidCharacters =  The name you will use to identify this report must only include letters a to z, numbers 0 to 9
+reportName.change.hidden = Report name
```

Comment:
Can we add these messages to messages.cy file as well with the "cy:" prefix. Please see [here](https://github.com/hmrc/trade-reporting-extracts-frontend/blob/main/conf/messages.cy) for examples.